builder-images/Dockerfile.template

```diff
@@ -28,6 +28,15 @@ RUN microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install -y apr
     && microdnf clean all \
     && rpm -q apr-devel autoconf automake bc buildah bzip2-devel cmake diffutils file findutils gcc gcc-c++ git glibc-devel glibc-devel.i686 glibc-langpack-en glibc-static gzip libcurl-devel libgcc.i686 libstdc++-static libtool lsof make openssl-devel podman shadow-utils tar unzip wget which zlib-devel
 
+RUN microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install -y emacs-filesystem vim-filesystem
+# XXX: RHEL 8 does not have ninja-build, so grabbing from CentOS 8
+RUN rpm --import https://www.centos.org/keys/RPM-GPG-KEY-CentOS-Official \
+    && rpm -q gpg-pubkey-8483c65d-5ccc5b19
+RUN wget --no-verbose https://vault.centos.org/centos/8/PowerTools/x86_64/os/Packages/ninja-build-1.8.2-1.el8.x86_64.rpm \
+    && rpm -K ninja-build-1.8.2-1.el8.x86_64.rpm \
```

Comment:
This checks the package signature.

Reply:
Ah, sorry, my bad

Reply:
Yeah, it was using GPG signatures, but I think I still needed to verify the fingerprint of the downloaded GPG key, so thanks.